Plash/AppDelegate.swift

```diff
@@ -247,10 +247,25 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 	func loadURL(_ url: URL?) {
 		webViewError = nil
 
-		guard let url = url else {
+		guard var url = url else {
 			return
 		}
 
+		// Sub out placeholder variables for actual values.
+
+		// Here we swap out [screenWidth] and [screenHeight] for their actual values.
```

Comment:
Can you move al the logic here to a separate method?

Reply:
👍